vuu-ui/packages/vuu-data/src/server-proxy/buffer-range.ts

```diff
@@ -1,4 +1,9 @@
 import { VuuRange } from "@finos/vuu-protocol-types";
+import { logger } from "@finos/vuu-utils";
+
+const loggingLevel = () => {
```

Comment:
this function is repeated in every file. I don't think we need it at all (see comment above) but if we wanted to keep it, it should be declared once and imported where needed.